include/experimental/__p1673_bits/linalg_config.h.in

```diff
@@ -4,3 +4,4 @@
 #cmakedefine LINALG_ENABLE_CONCEPTS
 #cmakedefine LINALG_ENABLE_KOKKOS
 #cmakedefine LINALG_ENABLE_KOKKOS_DEFAULT
+#cmakedefine LINALG_ENABLE_TESTS
```

Comment:
Why do you need it?

Reply:
I was using it inside the kokkos impl to conditionally make a print that is detected from the tests